.github/CONTRIBUTING.md

```diff
@@ -40,9 +40,24 @@ A bug must have an issue tracking it in the issue tracker that has been approved
 
 ## Contributing features
 
-We're happy to discuss schema proposals as long as they align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). 
+Thank you for wanting to contribute to Adaptive Cards. If you are planning to make changes to the Core SDK, please make sure these changes align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). Prior to writing any code that affects rendering, please make sure you follow the following steps:
+1.	Open a “Feature Request” issue to start the discussion on the feature you wish to see in the product. Please make sure you search our planned work to see if your feature is already planned. Please make sure you search our [planned work](https://portal.productboard.com/adaptivecards/1-adaptive-cards-features/tabs/6-planned) to see if your feature is already planned.
+2.	Open a PR with a .MD file to the [following directory](https://github.com/microsoft/AdaptiveCards/tree/main/specs/elements) with a spec that clearly outlines the feature you are implementing. **Include as much detail on the feature as possible**.
+    1.	At a bare minimum you should look to answer the following questions:
+        1.	Why do you need this feature and what will the feature do?
+        2.	What will the API surface look like?
+        3.	What will the schema look like?
+        4.	What is the down-level impact of this feature?
+    2.  Please make sure your PR follows a similar format as our [Spec](https://github.com/microsoft/AdaptiveCards/blob/main/specs/SpecTemplate.md) template including a summary, rationale, scope, and examples.
+3.	Once you open the PR with the spec, the team will review it and schedule a call with the author to walk through the feature. This is where the discussion will happen.
```

Comment:
Instead of `schedule a call with the author`, perhaps we can say `schedule a call with the community` to make it more inclusive for other folks who are interested